challenges/views-info.py

```diff
@@ -0,0 +1,16 @@
+from django.shortcuts import render
+from django.contrib import messages
+from django.views.generic import TemplateView
+from .forms import ContactForm
+
+def home(request):
+    form = ContactForm(request.POST)
+    if request.method == 'POST':
+        if form.is_valid():
+            form.save()
+            name = form.cleaned_data.get('name')
+            message = form.cleaned_data.get('message')
+            messages.success(request, f'Your Account has now been created for {username}! You can now log in!')

```

Comment:
`username` is not defined

Reply:
Whoops copied and pasted without editing from some of my other code. 

That should read:

messages.success(request, f'Your message has successfully sent')